libs/langchain-azure-openai/src/embeddings.ts

```diff
@@ -48,11 +48,9 @@ export class AzureOpenAIEmbeddings
 
     super(fieldsWithDefaults);
 
-    this.azureOpenAIEmbeddingsApiDeploymentName =
-      (fieldsWithDefaults?.azureOpenAIEmbeddingsApiDeploymentName ||
-        fieldsWithDefaults?.azureOpenAIApiDeploymentName) ??
-      (getEnvironmentVariable("AZURE_OPENAI_API_EMBEDDINGS_DEPLOYMENT_NAME") ||
-        getEnvironmentVariable("AZURE_OPENAI_API_DEPLOYMENT_NAME"));
+    this.azureOpenAIApiDeploymentName =
+      fieldsWithDefaults?.azureOpenAIApiDeploymentName ??
+      getEnvironmentVariable("AZURE_OPENAI_API_DEPLOYMENT_NAME");
```

Comment:
I actually think we want to keep this env var for embeddings and have it use the current behavior - it's apparently common for completions vs. embeddings endpoints.

Yes let's make the change on the input field name though.